idl-compiler.py

```diff
@@ -1434,6 +1443,18 @@ def add_view(cout, cls):
             }}
         """).format(f=DESERIALIZER, **locals()))
 
+        if is_vector(m.type):
+            item_type = basic_type(m.type)
```

Comment:
What if it's None?

Reply:
How can it be None if m.type is vector?

Reply:
I misread, somehow I thought the function ended before the is_vector() call, don't know why.